fuzz/fuzz_adt_priq.c

```diff
@@ -120,7 +120,7 @@ test_priq_operations(uintptr_t limit)
 			.enable = true,
 		},
 
-		.seed = theft_seed_of_time(),
+		.seed = seed,
```

Comment:
> The seed is passed along from main() to all test_*() functions. This means all those share the same seed. I wanted to mention that just to check that the PRNG is re-seeded for each invocation of theft_run().

Correct - if you set the seed here, it will use that seed for the _first_ trial, but subsequent trials will use different seeds derived from that seed somehow. IIRC, it re-seeds the PRNG with the last trial seed and draws a new random value for the next, but it could also just increment the seed or something. It's intentionally an implementation detail how seeds are chosen, besides the initial seed and "always" seeds, and may change in a future version.

The PRNG is currently not seeded to a consistent value during the property test function itself (because a `struct theft *t` handle is passed in, you can draw new random values during the property test, in case probabilistic algorithms are being used), but that's something I plan on fixing in the next release.